### PR TITLE
Raw detection output if measure = raw

### PR DESCRIPTION
**Pull request by fantes**
WIP : 
we need to decide how to pass the data wrt prediction/test   memorydatalayer/AnnotatedDataLayer

### CODE REVIEW

File: src/supervisedoutputconnector.h [hunk 1]
```diff
@@ -1542,7 +1549,71 @@ namespace dd
...
+      APIData bad = ad.getobj("0");
+      int pos_count = ad.get("pos_count").get<int>();
+      for (int i=0;i<pos_count;i++)
+        {
+          std::vector<APIData> vbad = bad.getv(std::to_string(i));
+          //std::cerr << "vbad size=" << vbad.size() << std::endl;
+          for (size_t j=0;j<vbad.size();j++)
+            {
+              std::vector<double> tp_d = vbad.at(j).get("tp_d").get<std::vector<double>>();
+              //std::cerr << "tp_d size=" << tp_d.size() << std::endl;
```
Comment (beniz) [inline src/supervisedoutputconnector.h]: Let's remove comments before merge! tks.

**Comment by beniz** [comment]
Setting `measure:["raw"]` is enough, either for training or predict calls ?

File: src/supervisedoutputconnector.h (hunk 1 above)
Comment (fantes) [reply src/supervisedoutputconnector.h]: done

**Comment by fantes** [comment]
> Setting `measure:["raw"]` is enough, either for training or predict calls ?

yes it is !

**Pull request merged**